judo-ui-react/src/main/resources/actor/src/components/widgets/NumericInput.tsx.hbs

```diff
@@ -5,6 +5,10 @@ import { decimalSeparators, thousandSeparator } from '~/l10n/separators';
 
 export function NumericInput<BaseType = InputAttributes>(props: NumericFormatProps<BaseType>) {
   const { locale: l10nLocale } = useL10N();
+  const propsExpanded: NumericFormatProps<BaseType> = {
+    decimalScale: 0,
+    ...props
```

Comment:
Does `...props` contain information if `decimalScale` is needed and in that case it overrides `decimalScale: 0`? Or am i missing something?

Reply:
ofc.